spec.bs

```diff
@@ -8375,69 +8378,65 @@ To <dfn>build trusted key value bidding signals request body</dfn> given a [=set
   1. If |compressionIdMap| does not [=map/contain=] |joiningOrigin|:
     1. [=map/Set=] |compressionIdMap|[|joiningOrigin|] to |nextCompressionGroupId|.
     1. Increment |nextCompressionGroupId| by 1.
   1. Let |compressionGroupId| be |compressionIdMap|[|joiningOrigin|].
   1. If |compressionGroupMap| does not [=map/contain=] |compressionGroupId|:
-    1. Let |compressionGroupMap|[|compressionGroupId|] be an empty [=map=], whose [=map/keys=] are integers
+    1. [=map/Set=] |compressionGroupMap|[|compressionGroupId|] to an empty [=map=], whose [=map/keys=] are integers
       and [=map/values=] are [=maps=].
-  1. Set |executionMode| to |group|'s [=interest group/execution mode=].
-  1. If |executionMode| equal to "`group-by-origin`", set |partitionId| to 0.
+  1. If |group|'s [=interest group/execution mode=] is "`group-by-origin`", set |partitionId| to 0.
   1. Otherwise:
     1. If |compressionGroupMap|[|compressionGroupId|] [=map/contains=] 0, set |partitionId| to
       [=map/size=] of |compressionGroupMap|[|compressionGroupId|].
-    1. Otherwise, set |partitionId| to sum [=list/size=] of |compressionGroupMap|[|joiningOrigin|]
-      and 1.
-  1. Set |interestGroupIdMap|[|group|'s [=interest group/name=]] to [=tuple=] of |compressionGroupId|
-    and |partitionId|.
-  1. If |compressionGroupMap|[|compressionGroupId|] does not [=map/contain=] |partitionId|:
-    1. Let |partition| be an empty [=partition=].
-    1. Set |partition|'s [=partition/id=] to |partitionId|.
-    1. Let |namespace| be an empty [=map=], whose [=map/keys=] are [=strings=] and [=map/values=] are [=list=]
-      of [=strings=].
-    1. Set |namespace|["interest_group_names"] to [|group|'s [=interest group/name=]].
-    1. Set |namespace|["bidding_keys"] to |group|'s [=interest group/trusted bidding signals keys=].
-    1. Set |partition|'s [=partition/namespace=] to |namespace|.
-    1. Let |metadata| be an empty [=map=], whose [=map/keys=] and [=map/values=] are [=strings=].
-    1. Set |metadata|["experiment_group_id"] to |experimentGroupId|.
-    1. Set |metadata|[|slotSizeParams|[0]] to |slotSizeParams|[1].
-    1. Set |partition|'s [=partition/metadata=] to |metadata|.
-    1. Set |compressionGroupMap|[|compressionGroupId|][|partitionId|] to |partition|.
+    1. Otherwise, set |partitionId| to [=list/size=] of |compressionGroupMap|[|joiningOrigin|] plus 1.
+  1. [=map/Set=] |interestGroupIdMap|[|group|'s [=interest group/name=]] to [=tuple=] of
+    (|compressionGroupId|, |partitionId|).
+  1. If |compressionGroupMap|[|compressionGroupId|] does not [=map/contain=] |partitionId|, set
+    |compressionGroupMap|[|compressionGroupId|][|partitionId|] to a [=bidding partition=] with the following [=struct/items=]:
+    : [=bidding partition/id=]
+    :: |partitionId|
+
+    : [=bidding partition/namespace=]
+    :: The [=ordered map=] «[ "interestGroupNames" → « [|group|'s [=interest group/name=]],

```

Comment:
```suggestion
    :: The [=ordered map=] «[ "interestGroupNames" → « |group|'s [=interest group/name=] »,
```

Reply:
[|group|'s [=interest group/name=]] is the preferred one since the value is a list, and |group|'s [=interest group/name=] is a single string, unless the format is incorrect.

Reply:
But there is a bad "«", removed it.

Reply:
why putting `|group|'s [=interest group/name=]` in a "[]"?

Reply:
Because |group|'s [=interest group/name=] is a single string, and the expected value is a list of strings

Reply:
spec uses `« »` for lists, not `[]`.